### PR TITLE
added notes to the readme

### PR DESCRIPTION
**Pull request by merit-maita**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-27466

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 1]
```diff
@@ -691,7 +691,10 @@ Retrieves a list of members from a channel.
 ### microsoft-teams-chat-create
 ***
 Creates a new chat. 
-Note: Only one oneOnOne chat can exist between two members. If a oneOnOne chat already exists, it will be returned.
+
+Notes: 
+- Only one oneOnOne chat can exist between two members. If a oneOnOne chat already exists, it will be returned.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:696]: Which chat is returned?
The new chat or the existing chat?

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 2]
```diff
@@ -743,6 +746,8 @@ Note: Only one oneOnOne chat can exist between two members. If a oneOnOne chat a
 ***
 Sends a new chat message in the specified chat.
 
+Note:
+- This command works with the consent user, not with the bot. Which means, the message is sent to the given chat by the consent user, not the bot.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:750]:
```suggestion
This command works with the consent user, not with the bot. Which means, the message is sent to the given chat by the consent user, not the bot.
```

**Comment by ShirleyDenkberg** [comment]
@michal-dagan @thefrieddan1 Doc review completed.

**Review by thefrieddan1 — APPROVED** [review]
LGTM

**Review by michal-dagan — APPROVED** [review]

**Pull request merged**